frontend/src/components/applications/review/OperatingCostsSummary.vue

```diff
@@ -14,29 +14,54 @@
       </div>
       <div v-if="isRentLease" class="mt-4">
         <h4>Uploaded Document(s)</h4>
-        <v-card v-if="isUploadedDocumentsComplete" class="my-1 pa-2" variant="outlined">
-          <div v-for="document in documents" :key="document.documentId" class="px-2 py-1">{{ document.fileName }}</div>
+        <v-card class="pa-3" variant="outlined">
+          <v-card class="mt-2 mb-3 pa-3">
+            <AppDocumentUpload
+              :readonly="true"
+              :documentType="DOCUMENT_TYPES.FINANCIAL_STATEMENT"
+              :uploadedDocuments="documentsFinancialStatements">
+              <AppMissingInfoError v-if="!documentsFinancialStatements.length" :to="{ name: 'operating-costs', hash: '#application-document-upload', params: { applicationGuid: $route.params.applicationGuid } }">
+                {{ DOCUMENT_TYPES.FINANCIAL_STATEMENT }} document upload required
+              </AppMissingInfoError>
+            </AppDocumentUpload>
+
+          </v-card>
+          <v-card class="pl-3">
+            <AppDocumentUpload class="pt-4 pr-3"
+              :readonly="true"
+              :documentType="DOCUMENT_TYPES.BALANCE_SHEET"
+              :uploadedDocuments="documentsBalanceSheets">
+              <AppMissingInfoError v-if="!documentsBalanceSheets.length" :to="{ name: 'operating-costs', hash: '#application-document-upload', params: { applicationGuid: $route.params.applicationGuid } }">
+                {{ DOCUMENT_TYPES.BALANCE_SHEET }} document upload required
+              </AppMissingInfoError>
+            </AppDocumentUpload>
+          </v-card>
+          <v-card class="pl-3 mt-3 pt-0">
+            <AppDocumentUpload class="pt-4"
+              :readonly="true"
+              :documentType="DOCUMENT_TYPES.SUPPORTING_DOCS"
+              :uploadedDocuments="documentsSupporting" />
```

Comment:
if the Facility Type is Rent/Lease, we will need to check if the SupportingDoc is not empty (i.e.: the providers need to upload the Rent/Lease document)

Reply:
please check if we can reuse/update this function isUploadedDocumentsComplete().

Reply:
function removed, no longer needed